README.md

```diff
@@ -138,6 +138,7 @@ Device notes are sorted alphabetically, major (upper case) notes before minor
 | AIO liquid cooler  | [NZXT Kraken X53, X63, X73](docs/kraken-x3-z3-guide.md) | <sup>_h_</sup> | 1.11.1 |
 | AIO liquid cooler  | [NZXT Kraken Z53, Z63, Z73](docs/kraken-x3-z3-guide.md) | <sup>_h_</sup> | 1.11.1 |
 | AIO liquid cooler  | ~~[NZXT Kraken 2023 Standard, Elite](docs/kraken-x3-z3-guide.md)~~ | <sup>_B_</sup> | git |
+| AIO liquid cooler  | [NZXT Kraken 2024 Elite RGB](docs/kraken-x3-z3-guide.md) | <sup>_B_</sup> | git |
```

Comment:
@jonasmalacofilho Can you please remind me what were the issues with the 2023 models? I remember it was something about images, but for the life of me can't find the issue. Trying to ascertain if this legend is correct, or should also be striked out. Thanks!

Reply:
I sort of lost track too. But the issue seems to be about [animated] GIF on firmware 2.x.

https://github.com/liquidctl/liquidctl/blob/702bf32f2b9e/liquidctl/driver/kraken3.py#L822C1-L825C18

---

On a related note, it's weird that from the few device families with the [B]roken tag, only the Krakens are in ~~strikethrough~~. I can't find or remember any reason why I would have intentionally left it that way, so maybe it was accidental.

Reply:
The 2024 RGB uses the 1.2.0 firmware, and it's considered up to date. Whether this means it's using the same branch but older version is a mystery to me, but GIF images work fine.